share/functions/cd.fish

```diff
@@ -3,10 +3,17 @@
 #
 function cd --description "Change directory"
     set -l MAX_DIR_HIST 25
+    set -l options h/help L P
+    argparse -n cd --exclusive L,P --max-args=3 $options -- $argv
```

Comment:
The `--max-args` argument should be two because it only restricts the number of non-option arguments, including the command name itself.

```fish
$ argparse -n cd --max-args=2 --exclusive L,P h/help L/logical P/physical -- cd -L $PWD
$ argparse -n cd --max-args=2 --exclusive L,P h/help L/logical P/physical -- cd -L $PWD $PWD
cd: Expected at most 2 args, got 3
```

Reply:
It should be *1*, because the command name isn't in $argv.

Reply:
Sorry yes, I will fixed it.